### DISCUSSION TITLE
SMITE 1.0.3A BOOT BUG

### DISCUSSION THREAD

**Discussion by AnonyAce** [thatskymod/Sky-CotL-Scripts#46] [Q&A]
Whenever I try to open Canvas with smite 1.0.3a It closes the app after receiving this error message:
java.lang.Exception: đlopen failed: tibrary "libcpr.so" not
foand: needed by /data/data/git.artđeell.skymođloader/files/
mods/libsmite.so in namespace classloader-namespace
at
git.artdeell.skymođloader.LibrarySelectorListener.onModli
brary(Native Method
at
git.artdeell.skymođloađer.elfmod.EfRefcoantL oader.loadNa
tivelllfRefcoantLoader.java:57
at
git.artđeell.skymođloađer.EffLoađer.loadLib(EtfLoađer.java:
69)
at
git.artdeell.skymodloader.elfmod.EfRefcoantL.oader.oad(E1
fRefcomtLoađer.java:48)
at
git.artđeell.skymodloađer.Main@ctivity.loadGame(Maingctiv
ityjava:)
at
git. artdeell.skymodloađer. Main@ctivity.find@ndLoad(Maing
ctivityjava:49)
at
git.artđeell.skymođloađer.Main@ctivity.onCreate(Maingctivi
ty.java33)
at android.app.@ctivity.performCreate(@ctivity.java:8591)
at android.app.@ctivity.performCreate(@ctivityjava:8570)
at
android.app.Instramentation.call@ctivityOnCreate(instramen
tation.java:88)
at
anđroiđ.app.@ctivity Tbread.performLaanchActivity(@ctivity
Thread.java:4150)
at
android.app.@ctivityTbread.bandleLaancb@ctivitg(@ctivityT
breadjava:4325)
at
android.app.servertransaction.LaanchActivityItem.execate(
LaandbActivityItemjava:to)
at
anđroiđ.app.servertransaction. TransactionExecator.execate
Callbacks(TransactionE xecator.java:35)
at
android.app.servertransaction. TransactionExecator.execate
(TransactionE.xecatorjava:9")
at
android.app.@ctivit¥ TbreadsI.bandleMessage(ActivityTbre
ad,java:257)
at android.os.ITanđler.dispatciMessage(Ianđler.java:1o6)
at android.os.Looper.loopOnce(ooper.java:226)
at android.os.L ooper.loop(L ooper.java:13)
at
anđroiđ. app.(ctivityTbread.main(gctivity Thread.java:8757)
at java.lang.reflect.Method.invoke(Native Mettoo
at
com.android.internal.os.RontimeInitsMetbod@nd@rgsCaller.r
an(Rantimelnit.java5r)
at
com.anđroiđ.internal.os.zxgotelnit.main(Zvgoteľnit.java:/o67)
OK

**Comment by thatskymod** [comment]
### Issue:
The error message you are seeing indicates that the library 'libcpr.so', required by libsmite 1.0.3a, could not be found. This is due to a naming inconsistency between Canvas and libsmite itself.

### Solution:
Please upgrade to [libsmite-1.0.4a.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libsmite-1.0.4a.so) and try installing the mod again. Doing so should eliminate the error, allowing the mod to be successfully added.

**Reply by AnonyAce** [reply]
Thank you so much!
